Amazon.IonObjectMapper.Test/TestCustomSerializersFactory.cs

```diff
@@ -52,7 +52,7 @@ public int AddSections(int sections)
 
         public string ShowCurrentTime ()
         {
-            return DateTime.Now.ToString();
+            return DateTime.Parse("2021-10-10T13:15:21Z").ToString();
```

Comment:
Similar, currently this goes from string -> DateTime -> string, you can return just the string directly.

Reply:
Changed to use string directly.